crates/bevy_mesh/src/index.rs

```diff
@@ -37,7 +37,7 @@ where
 #[derive(Debug, Error)]
 pub enum MeshWindingInvertError {
     /// This error occurs when you try to invert the winding for a mesh with [`PrimitiveTopology::PointList`](super::PrimitiveTopology::PointList).
-    #[error("Mesh winding invertation does not work for primitive topology `PointList`")]
+    #[error("Mesh winding inversion does not work for primitive topology `PointList`")]
```

Comment:
Suggested in https://github.com/bevyengine/bevy/pull/16702#issuecomment-2525354642